### PR TITLE
Add From Nickname to sign

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot 2023-10-09 at 15 34 31
callSC
transaction
executeSc

### CODE REVIEW

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx
Comment (Thykof) [inline wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:1]: 😻

**Review by Thykof — APPROVED** [review]

File: wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx [hunk 1]
```diff
@@ -19,33 +20,39 @@ export function PlainText(props: PromptRequestData) {
...
-        </div>
+        <>
+          <hr className="h-0.25 bg-neutral opacity-40 w-full" />
+
+          <div className="flex flex-col w-full h-fit">
+            <AccordionCategory
+              isChild={false}
+              iconOpen={<FiChevronDown />}
+              iconClose={<FiChevronUp />}
+              customClass={'!p-0'}
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx:40]: Do we need to force here!? It should not be used.

Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx:40]: Screenshot from 2023-10-10 10-36-51
for some odd reason yes

Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx:40]: Screenshot from 2023-10-10 10-39-06
the custom class is set correctly too

Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx:40]: but setting it to px-0 and py-0 seems to work correctly

Comment (mazmassa) [reply wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx]: Better! 
Force with `!` should be avoided. ;)

Tks for that!

**Review by mazmassa — APPROVED** [review]

**Pull request merged**